### DISCUSSION TITLE
How do I add other federated feeds to my new mastodon server

### DISCUSSION THREAD

**Discussion by kmages** [mastodon/mastodon#20424] [Q&A]
I want my new Mastodon server (ver.news) to have a federated view of all toots. How do I do this?

**Comment by markcellus** [comment]
Looks like this was posted a second time https://github.com/mastodon/mastodon/discussions/20425.

A Mastodon dev can correct me if I'm wrong. But I think someone on your instance has to follow someone from another instance. Then that user's posts from the other instance will then begin to show up in the federated timeline on your instance. I'm not aware of any way to manually add instances to the federated timeline. 

Hope this helps!

**Comment by kmages** [comment]
I wish there was a script or some database to simply add.

On Fri, Nov 11, 2022 at 11:29 AM Mark ***@***.***> wrote:

> Looks like this was posted a second time #20425
> <https://github.com/mastodon/mastodon/discussions/20425>.
>
> A Mastodon dev can correct me if I'm wrong. But I think someone on your
> instance has to follow someone from another instance for that user's posts
> to show up in the federated timeline of your instance. I'm not aware of any
> way to manually add instances to the federated timeline.
>
> Hope this helps!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mastodon/mastodon/discussions/20424#discussioncomment-4119477>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACR26TFH6PKXCHU33BF46MLWHZ7ARANCNFSM6AAAAAAR5W2YBE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
kgm

Spelling and grammar curiosities courtesy of my mobile phone.

KenMages.com
